src/AlertBox/AlertBox.jsx

```diff
@@ -42,8 +42,8 @@ export default class AlertBox extends PureComponent {
     return (
       <div className={classnames(`AlertBox--${type}`, CLASSNAMES.CONTAINER, className)}>
         <div className={CLASSNAMES.HEADER}>
```

Comment:
Q. Why don't clever components use themselves? It seems like instead of using CSS we should use https://clever.github.io/components/#/components/flex-box ? Thoughts?

Reply:
Fair point! I've generally just viewed the `FlexBox` component as a convenience. And this case felt simple enough to just use CSS directly

Reply:
Let's FlexBox it up! I think it'll be cleaner code. :)

Reply:
That said, I'm realizing that, even if I don't use the `<FlexBox>` component, I should use the [flex mixins](https://github.com/Clever/components/blob/25fc1c1cef0d16391baebee1200aabc321b81ac9/src/less/flex.less#L11-L16) to ensure compatibility

Reply:
Or if you think the lift is too much.... then LGTM

Reply:
@arsalansufi if you're going down that route... then I would definitely use FlexBox. Way more maintainable.